resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -47,6 +48,17 @@ class ExecutorPodsAllocator(
 
   private val EXECUTOR_ID_COUNTER = new AtomicInteger(0)
 
+  private val PVC_COUNTER = new AtomicInteger(0)
+
+  private val maxPVCs = if (Utils.isDynamicAllocationEnabled(conf)) {
+    conf.get(DYN_ALLOCATION_MAX_EXECUTORS)
+  } else {
+    conf.getInt(EXECUTOR_INSTANCES.key, DEFAULT_NUMBER_EXECUTORS)
+  }
+
+  private val reusePVC = conf.get(KUBERNETES_DRIVER_OWN_PVC) &&
+    conf.get(KUBERNETES_DRIVER_REUSE_PVC) && conf.get(KUBERNETES_DRIVER_WAIT_TO_REUSE_PVC)
```

Comment:
Hmm, if don't use `KUBERNETES_DRIVER_WAIT_TO_REUSE_PVC`, cannot we still reuse PVCs?

Reply:
I think this named `reusePVC` is a bit confused. Isn't it the combination of three configs you mentioned (https://github.com/apache/spark/pull/38943/files#r1041347366)?

Maybe `podAllocOnPVC`?

Reply:
You are right. Let me rename~